### PR TITLE
Fix formating of settle command in UG

### LINKED ISSUE #222: Fix formatting of settle command in Ug

### PR DESCRIPTION
**Pull request by Soumyadip-cmd**
Closes #222

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -225,17 +225,20 @@ In case you accidentally make a mistake using the <md>`owe`</md> command, you ca
...
-Examples:<br>`settle 1 amount/500.00`
+**Examples:**
+* `settle 1 amount/500.00`
+
+**Output:**
 
-![settleResult.png](images%2FsettleResult.png)
+![settleResult.jpg](images%2FsettleResult.jpg)
 
 <box type="important" header="#### Constraints">
```
Comment (Feng1231) [inline docs/UserGuide.md:237]: This one need 5 # right

**Review by Feng1231 — COMMENTED** [review]
I think the box's header need to be 5 `#`

File: docs/UserGuide.md (hunk 1 above)
Comment (Soumyadip-cmd) [reply docs/UserGuide.md]: Thank you for pointing that out. I have pulled the changes into this branch.

**Review by TY1Fan — APPROVED** [review]
LGTM

**Pull request merged**